infrastructure/src/test/java/org/corfudb/infrastructure/management/ClusterGraphTest.java

```diff
@@ -30,142 +34,146 @@ public void testTransformation() {
         NodeState a = NodeState.builder()
                 .sequencerMetrics(SequencerMetrics.READY)
                 .heartbeat(new HeartbeatTimestamp(0, 0))
-                .connectivity(connectivity("a", ImmutableMap.of("a", OK, "b", OK, "c", FAILED)))
+                .connectivity(connectivity(A, ImmutableMap.of(A, OK, B, OK, C, FAILED)))
```

Comment:
Why change the test naming convention?

Reply:
don't get your question

Reply:
I am sorry for the confusion. I mean why change "a" to A? Other tests seems still use "a" instead of A.

Reply:
'A' is just a constant in `NodeStateTestUtil`. The node name is still lower case.
Don't want to repeat string names everywhere, so I decided to bring constants. Also, I can't replace old string names everywhere at one shot, will be doing that gradually